worker/worker.py

```diff
@@ -322,20 +321,40 @@ def heartbeat(worker_info, password, remote, current_state):
         print("Heartbeat stopped")
 
 
-def create_lock_file():
+def read_int(file):
+    try:
+        if path.exists(file):
```

Comment:
`path.exists()` could be dropped, there is already at work the exception `FileNotFoundError`.

Reply:
You are right! I fixed it.